commands2/__init__.py

```diff
@@ -1,96 +1,87 @@
-from . import _init_impl
-
-from .version import version as __version__
-
-# autogenerated by 'robotpy-build create-imports commands2 commands2._impl'
-from ._impl import (
-    Command,
-    CommandBase,
-    CommandGroupBase,
-    CommandScheduler,
-    ConditionalCommand,
-    FunctionalCommand,
-    InstantCommand,
-    MecanumControllerCommand,
-    NotifierCommand,
-    PIDCommand,
-    PIDSubsystem,
-    ParallelCommandGroup,
-    ParallelDeadlineGroup,
-    ParallelRaceGroup,
-    PerpetualCommand,
-    PrintCommand,
-    ProfiledPIDCommand,
-    ProfiledPIDSubsystem,
-    ProxyScheduleCommand,
-    RamseteCommand,
-    RepeatCommand,
-    RunCommand,
-    ScheduleCommand,
-    SelectCommand,
-    SequentialCommandGroup,
-    StartEndCommand,
-    Subsystem,
-    SubsystemBase,
-    Swerve2ControllerCommand,
-    Swerve3ControllerCommand,
-    Swerve4ControllerCommand,
-    Swerve6ControllerCommand,
-    TimedCommandRobot,
-    TrapezoidProfileCommand,
-    TrapezoidProfileCommandRadians,
-    TrapezoidProfileSubsystem,
-    TrapezoidProfileSubsystemRadians,
-    Trigger,
-    WaitCommand,
-    WaitUntilCommand,
-    # button,
-    # cmd,
-    requirementsDisjoint,
+from . import button
+from .command import Command, InterruptionBehavior
+from .commandfactories import (
```

Comment:
Why have the command factories moved modules?

Reply:
I organized it in a separate file as Java did. I didn't want to clutter `__init__.py` with them so I defined them in a separate file and reexported everything so you can still access them directly like in 2023.

Reply:
My point is they were already in a module but you changed its name.